tiled_layers/export-tiles/src/main/java/com/esri/samples/export_tiles/ExportTilesSample.java

```diff
@@ -105,97 +110,102 @@ public void start(Stage stage) {
+              // disable the button and show the progress bar
+              exportTilesButton.setDisable(true);
+              progressBar.setVisible(true);
+
+              // create a file and define the scale for the job
+              File tempFile = File.createTempFile("tiles", ".tpkx");
+              tempFile.deleteOnExit();
+
+              // create a new export tile cache task
+              var exportTileCacheTask = new ExportTileCacheTask(tiledLayer.getUri());
+
+              // create parameters for the export tiles job
+              double mapScale = mapView.getMapScale();
+              // the max scale is parameter is set to 10% of the map's scale to limit the
+              // number of tiles exported to within the tiled layer's max tile export limit
+              ListenableFuture<ExportTileCacheParameters> exportTileCacheParametersListenableFuture =
+                exportTileCacheTask.createDefaultExportTileCacheParametersAsync(downloadArea.getGeometry(), mapScale, mapScale * 0.1);
+
+              exportTileCacheParametersListenableFuture.addDoneListener(() -> {
+                try {
+                  var exportTileCacheParameters = exportTileCacheParametersListenableFuture.get();
+
+                  // create a job with the parameters
+                  var exportTileCacheJob = exportTileCacheTask.exportTileCache(exportTileCacheParameters, tempFile.getAbsolutePath());
+
+                  // start the job and wait for it to finish
+                  exportTileCacheJob.start();
+                  exportTileCacheJob.addProgressChangedListener(() -> progressBar.setProgress(exportTileCacheJob.getProgress() / 100.0));
+                  exportTileCacheJob.addJobDoneListener(() -> {
+                    if (exportTileCacheJob.getStatus() == Job.Status.SUCCEEDED) {
+
+                      // show preview of exported tiles in alert window
+                      TileCache tileCache = exportTileCacheJob.getResult();
+                      Alert preview = new Alert(Alert.AlertType.INFORMATION);
+                      preview.initOwner(mapView.getScene().getWindow());
+                      preview.setTitle("Preview");
+                      preview.setHeaderText("Exported to " + tileCache.getPath());
+                      MapView mapPreview = new MapView();
+                      mapPreview.setMinSize(400, 400);
+                      ArcGISTiledLayer tiledLayerPreview = new ArcGISTiledLayer(tileCache);
+                      ArcGISMap previewMap = new ArcGISMap(new Basemap(tiledLayerPreview));
+                      mapPreview.setMap(previewMap);
+                      preview.getDialogPane().setContent(mapPreview);
+                      preview.show();
+
+                    } else {
+                      Alert alert = new Alert(Alert.AlertType.ERROR, exportTileCacheJob.getError().getAdditionalMessage());
+                      alert.show();
```

Comment:
You could cut the alerts down to:
`new Alert(Alert.AlertType.ERROR, exportTileCacheJob.getError().getAdditionalMessage()).show()`

I see on line 205 that's been done so maybe tidy up across them all?